app/services/CheckoutService.scala

```diff
@@ -121,17 +122,22 @@ class CheckoutService(
     val idMinimalUser = authenticatedUserOpt.map(_.user)
 
     val telephoneNumber = subscriptionData.productData match {
-      case Left(paperData) => NormalisedTelephoneNumber.fromStringAndCountry(personalData.telephoneNumber, paperData.deliveryAddress.country orElse personalData.address.country)
+      case Left(paperData) => NormalisedTelephoneNumber.fromStringAndCountry(personalData.telephoneNumber, personalData.address.country orElse paperData.deliveryRecipient.address.country)
       case _ => NormalisedTelephoneNumber.fromStringAndCountry(personalData.telephoneNumber, personalData.address.country)
     }
 
+    val readerType: ReaderType = subscriptionData.productData match {
+      case Left(paperData) if paperData.deliveryRecipient.isGiftee => Gift
```

Comment:
So technically we could do gifts for Voucher+HD but we have decided not to offer it yet?

Reply:
Correct, the backend elements of this change will work for any paper product. The restriction/filter is limited to frontend elements.

Reply:
Cool, thanks for confirming 👍